### ISSUE TITLE
Handles dynamic notifications poorly.

### ISSUE THREAD

**Issue by wenottingham** [holylobster/nuntius-linux#9]
I use Glympse on my phone. It has a notification that has a time display in it that's updated every second (or so.)

If nuntius is enabled, I get a new notification on my desktop each time that the time is updated in the Android notification. In this case, that's every one to two seconds.

Ideally, nuntius would have a single notification that it updates the text of (like the Android notification) rather than constantly popping up new desktop notifications.

**Comment by nacho** [comment]
Yes I agree, we definitely need to improve this situation though as we pointed out this is still in a very early stage and we are working to make it better.
